src/containers/Tag/TagAdd/TagCheckbox.module.css

```diff
@@ -0,0 +1,12 @@
+.Input {
+    width: 100%;
+    height: 80px;
+	padding: 10px;
```

Comment:
Code is not indented.

Reply:
My bad about that! For some reason, I think Github does something to my indentation because on my end everything looks fine. Nevertheless, I'm going to re-space them to hopefully fix this issue.